server/core/src/main/scala/org.apache.daffodil.debugger.dap/DAPodil.scala

```diff
@@ -399,7 +404,7 @@ class DAPodil(
         for {
           variable <- launched.debugee.eval(args)
           _ <- variable match {
-            case None => session.sendResponse(request.respondFailure())
+            case None => session.terminate("eval failed")
```

Comment:
This might be something we should support in the future. Because currently even without the terminate when it send `respondFailure` the session is ended and the terminal is stopped. So this might be something to look into.

Reply:
@mbeckerle That makes sense to me, thanks for this information. I have created issue https://github.com/apache/daffodil-vscode/issues/671 from this comment. Feel free to add to the ticket if you thing anything is missing.